src/common/utility_unix.cpp

```diff
@@ -98,6 +111,16 @@ void Utility::setLaunchOnStartup(const QString &appName, const QString &guiName,
     }
 }
 
+#ifdef Q_OS_LINUX
+QString Utility::appImageLocation() {
+    return qEnvironmentVariable("APPIMAGE");
```

Comment:
Not sure about the performance nowadays but in the past we cached the env var in a local static.

Reply:
Seems a bit overkill. The method is called just once.